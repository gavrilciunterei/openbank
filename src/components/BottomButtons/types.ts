export type BottomButtonsProps = {
  children: React.ReactNode;
};
