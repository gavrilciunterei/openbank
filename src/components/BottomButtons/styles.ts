import styled from 'styled-components';

export const ButtonsContainer = styled.div`
  display: flex;
  justify-content: space-between;
  flex-direction: row-reverse;
`;

export const Line = styled.hr`
  border-top: 1px solid #f6f6f6;
`;
