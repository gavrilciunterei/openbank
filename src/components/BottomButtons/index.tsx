import React from 'react';
import ContentContainer from '../ContentContainer';
import { ButtonsContainer, Line } from './styles';
import { BottomButtonsProps } from './types';

function BottomButtons({ children }: BottomButtonsProps) {
  return (
    <div>
      <Line />
      <ContentContainer>
        <ButtonsContainer>{children}</ButtonsContainer>
      </ContentContainer>
    </div>
  );
}

export default BottomButtons;
