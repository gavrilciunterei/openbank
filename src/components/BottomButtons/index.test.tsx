import React from 'react';
import renderer from 'react-test-renderer';
import BottomButtons from './index';

it('Test BottomButtons', () => {
  const tree = renderer
    .create(
      <BottomButtons>
        <div>
          <button> botón 1 </button>
          <button> botón 2 </button>
        </div>
      </BottomButtons>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
