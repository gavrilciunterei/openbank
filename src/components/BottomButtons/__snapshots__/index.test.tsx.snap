// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test BottomButtons 1`] = `
<div>
  <hr
    className="sc-dkPtRN khVdif"
  />
  <div
    className="sc-bdvvtL gMSJIZ"
  >
    <div
      className="sc-gsDKAQ KHewZ"
    >
      <div>
        <button>
           botón 1 
        </button>
        <button>
           botón 2 
        </button>
      </div>
    </div>
  </div>
</div>
`;
